IntelliTect.TestTools.Selenate.Tests/ElementHandlerTests.cs

```diff
@@ -324,7 +326,97 @@ public void GetAttributeThrowsOnTimeout()
 
             var element = SetupElementHandler(mockDriver.Object);
 
-            Assert.Throws<WebDriverTimeoutException>(() => element.GetAttribute("Test"));
+            var ex = Assert.Throws<WebDriverTimeoutException>(() => element.GetAttribute("Test"));
+            Assert.Equal(typeof(NoSuchElementException), ex.InnerException?.GetType());
+        }
+
+        [Fact]
+        public void ChainElementFindsReturnsChildElement()
```

Comment:
The name of this test is not clear to me exactly what is being tested. It seems like SetupElementHandler uses the FindElement methods?

Reply:
Yeah, specifically this test is checking that if you do something like...

Driver.FindElement(By.Id("blah")).FindElement(By.Id("test")) then you get back that second element of ID "test". It's essentially checking that my wrapping isn't ultimately modifying the underlying Selenium behavior.

I could rename it to something like FindElementFromElementReturnsChild()?

Reply:
Yea I think that would have made more sense when I read it.